lib/vsc/utils/affinity.py

```diff
@@ -238,7 +238,7 @@ def sched_getcpu():
 #    tobin not used anymore
 def tobin(s):
     """Convert integer to binary format"""
-    ## bin() missing in 2.4
+    # bin() missing in 2.4
```

Comment:
we don't need to support python2.4 anymore, so actually this tobin can be dropped?